grails-app/services/org/pih/warehouse/report/NotificationService.groovy

```diff
@@ -172,7 +172,11 @@ class NotificationService {
         recipientItems.each { Person recipient, items ->
             if (emailValidator.isValid(recipient?.email)) {
                 def subject = g.message(code: "email.yourItemReceived.message", args: [shipment.destination.name, shipment.shipmentNumber])
-                def body = "${g.render(template: "/email/shipmentItemReceived", model: [shipmentInstance: shipment, receiptItems: items, recipient: recipient])}"
+                items.each {
+                    def quacan = it.quantityToCancel
+                    def co = quacan
```

Comment:
Can you explan these two lines?

Reply:
That's a great question! I had some issued with calculating quantity canceled by user in receipt before I realized that the only option to cancel is to cancel quantity remaining. So this part was only for me to debug.